snow/engine/common/tracker/accepted.go

```diff
@@ -49,24 +55,27 @@ func (a *accepted) OnValidatorRemoved(nodeID ids.NodeID, _ uint64) {
 	defer a.lock.Unlock()
 
 	a.validators.Remove(nodeID)
-	delete(a.frontier, nodeID)
+	delete(a.lastAccepted, nodeID)
 }
 
 func (*accepted) OnValidatorWeightChanged(_ ids.NodeID, _, _ uint64) {}
 
-func (a *accepted) SetLastAccepted(nodeID ids.NodeID, frontier ids.ID) {
+func (a *accepted) SetLastAccepted(nodeID ids.NodeID, frontier ids.ID, height uint64) {
```

Comment:
nit: should we rename `frontier` to `blockID` or just `id`?

Reply:
`id` it is!